e2e/nested-router-tab-view/package.json

```diff
@@ -6,22 +6,22 @@
   "nativescript": {
     "id": "org.nativescript.nestedroutertabview",
     "tns-ios": {
-      "version": "next"
+      "version": "6.0.0-2019-05-16-165318-01"
```

Comment:
leave this as `next`

Reply:
Yes, this was incorrect commit caused by CLI auto changing this tag. I have reverted it.